### PR TITLE
feature: increase text size for svgs

### PR DESCRIPTION
**Pull request by ivaylonikolov7**
- change "No randomness assigned" with "?"
- increase text font size for svgs

### CODE REVIEW

File: batch-nft-reveal/contracts/NFTCollection.sol [hunk 1]
```diff
@@ -202,13 +202,35 @@ contract NFTCollection is
...
+                abi.encodePacked(
+                    slicedRandomness[0],
+                    slicedRandomness[1],
+                    slicedRandomness[2],
+                    slicedRandomness[3],
+                    slicedRandomness[4],
+                    slicedRandomness[5],
+                    slicedRandomness[6]
+                )
+            );
```
Comment (imollov) [inline batch-nft-reveal/contracts/NFTCollection.sol:230]: Isn't there a way to do all this with style props?

File: batch-nft-reveal/contracts/NFTCollection.sol [hunk 2]
```diff
@@ -202,13 +202,35 @@ contract NFTCollection is
         returns (string memory)
     {
         string[3] memory parts;
-
-        parts[0] = '<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMinYMin meet" viewBox="0 0 350 350"><style>.base { fill: white; font-family: serif; font-size: 8px; }</style><rect width="100%" height="100%" fill="black" /><text x="10" y="20" class="base">';
-
         if (_metadataCleared) {
-            parts[1] = Strings.toString(_randomness);
+            parts[0] = '<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMinYMin meet" viewBox="0 0 350 350"><style>.base { fill: white; font-family: serif; font-size: 8px; }</style><rect width="100%" height="100%" fill="black" /><text class="base">';
```
Comment (imollov) [inline batch-nft-reveal/contracts/NFTCollection.sol:206]: Please move the common part on top.

File: batch-nft-reveal/contracts/NFTCollection.sol [hunk 3]
```diff
@@ -315,4 +337,17 @@ contract NFTCollection is
     function setRevealInterval(uint256 _revealInterval) external onlyOwner {
         s_revealInterval = _revealInterval;
     }
+
+    function substring(
```
Comment (imollov) [inline batch-nft-reveal/contracts/NFTCollection.sol:341]: Why is this function in the setters section?

File: batch-nft-reveal/contracts/NFTCollection.sol (hunk 1 above)
Comment (imollov) [inline batch-nft-reveal/contracts/NFTCollection.sol:230]: Please format these lines with prettier.

**Comment by imollov** [comment]
@ivaylonikolov7 pls revert these changes: https://github.com/hackbg/chainlink-keepers-templates/compare/2b1dae11504f1e80668485a879155277da07ccac..21c5178e66d3324bdac3a5f66111fea3c2e69df9

File: batch-nft-reveal/contracts/NFTCollection.sol [hunk 4]
```diff
@@ -315,4 +351,19 @@ contract NFTCollection is
     function setRevealInterval(uint256 _revealInterval) external onlyOwner {
         s_revealInterval = _revealInterval;
     }
+
+    // UTILITIES
```
Comment (imollov) [inline batch-nft-reveal/contracts/NFTCollection.sol:355]: What's the difference with helpers?

File: batch-nft-reveal/test/NFTCollection.test.ts [hunk 5]
```diff
@@ -185,7 +185,7 @@ describe("NFTCollection", async function () {
         "./test/data/revealed_metadata.txt",
         "utf8"
       );
-      const uri = await nftCollection.tokenURI(0);
+      const uri = await nftCollection.tokenURI(1);
```
Comment (imollov) [inline batch-nft-reveal/test/NFTCollection.test.ts:188]: Why the other test `should show unrevealed message` is querying token with id `0`?

Comment (ivaylonikolov7) [reply batch-nft-reveal/test/NFTCollection.test.ts:188]: When you `tokenUri` nonexistent token it shows the unrevealed message. We need to add a `require` when you ask for nonexistent token's tokenUri.

File: batch-nft-reveal/test/NFTCollection.test.ts [hunk 6]
```diff
@@ -172,7 +172,8 @@ describe("NFTCollection", async function () {
         "./test/data/unrevealed_metadata.txt",
         "utf8"
       );
-      const uri = await nftCollection.tokenURI(0);
+      await mint(nftCollection, NFT_REVEAL_BATCH_SIZE);
```
Comment (imollov) [inline batch-nft-reveal/test/NFTCollection.test.ts:175]: The first part of a unit test is to do the setup. Check the other tests. Also add new lines like the other tests for consistency.

**Review by imollov — APPROVED** [review]

**Pull request merged**